cinder/volume/drivers/nexenta/ns5/jsonrpc.py

```diff
@@ -149,7 +149,31 @@ def handle_failover(self):
             self.__proxy.__init__(
                 host, self.__proxy.port, self.__proxy.user,
                 self.__proxy.password, self.__proxy.use_https,
-                self.__proxy.verify)
+                self.__proxy.pool, self.__proxy.verify)
+            url = self.get_full_url('rsf/clusters')
+            response = self.__proxy.session.get(
+                url, verify=self.__proxy.verify)
+            content = response.json() if response.content else None
+            cluster_name = content['data'][0]['clusterName']
```

Comment:
This line crashes the program if 'content' is None that can happen at previous line